src/components/cards/Cards.js

```diff
@@ -30,7 +31,8 @@ function Cards() {
 
 export default Cards;
 
-/* --- old cards
+/* ------ former "cards.js" before import via "data.json" --------
+
 
 function Cards() {
   return (
```

Comment:
I do understand that for reference it is nice to still have this commented out code, however, this is what git is best at! Even if you remove it, you have commited it and you can check it out in one of your previous commits. This way you can keep you main branch clean of all commented out code.